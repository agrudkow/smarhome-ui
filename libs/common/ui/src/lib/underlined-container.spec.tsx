import React from 'react';
import { render } from '@testing-library/react';

import UnderlinedContainer from './underlined-container';

describe(' UnderlinedContainer', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<UnderlinedContainer />);
    expect(baseElement).toBeTruthy();
  });
});
