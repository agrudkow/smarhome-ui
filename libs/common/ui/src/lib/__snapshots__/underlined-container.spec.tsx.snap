// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` UnderlinedContainer should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  border-bottom: 2px solid #1BBDCE;
}

@media (max-width:480px) {
  .c0 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
    -webkit-box-pack: start;
    -webkit-justify-content: flex-start;
    -ms-flex-pack: start;
    justify-content: flex-start;
    -webkit-align-items: flex-start;
    -webkit-box-align: flex-start;
    -ms-flex-align: flex-start;
    align-items: flex-start;
  }
}

<div>
  <div
    class="c0"
  />
</div>
`;
